### PR TITLE
Clean up unused definitions and other respec errors

### PR DESCRIPTION
**Pull request by yoavweiss**
***
<a href="https://pr-preview.s3.amazonaws.com/yoavweiss/resource-timing/pull/310.html" title="Last updated on Jan 12, 2022, 4:05 AM UTC (2367c71)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/resource-timing/310/c6ad017...yoavweiss:2367c71.html" title="Last updated on Jan 12, 2022, 4:05 AM UTC (2367c71)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -212,38 +212,9 @@ <h2>
...
-        being retrieved (such as by author script), and is said to be
-        <dfn>setting</dfn> when a new value is assigned to it.
-      </p>
-      <p>
-        The term <dfn>JavaScript</dfn> is used to refer to ECMA262, rather than
-        the official term ECMAScript, since the term JavaScript is more widely
-        known. [[ECMASCRIPT]]
-      </p>
-      <p>
-        The term <dfn>resource</dfn> is used to refer to elements and any other
```
Comment (npm1) [inline index.html:232]: It seems fine to delete this but does this API still have a clear explanation about what resources are exposed via ResourceTiming?

File: index.html [hunk 2]
```diff
@@ -212,38 +212,9 @@ <h2>
...
-        user-initiated fetches throughout this specification. For example, a
-        resource could originate from XMLHttpRequest objects [[XHR]], HTML
-        elements [[HTML]] such as iframe, img, script, object, embed, and link
-        with the link type of stylesheet, and SVG elements [[SVG11]] such as
-        svg.
-      </p>
       <p>
         The term <dfn>cross-origin</dfn> is used to mean non [=same origin=].
```
Comment (npm1) [inline index.html:216]: Is this needed? It seems like something that ought to be defined elsewhere.

File: index.html [hunk 3]
```diff
@@ -504,71 +475,72 @@ <h3>
...
             <li>
-              <dfn>"navigation"</dfn>, if the request is a [=navigation
+              <code>"navigation"</code>, if the request is a [=navigation
               request=];
             </li>
             <li>
-              <dfn>"css"</dfn>, if the request is a result of processing a CSS
-              <a data-xref-type="css-function">url()</a> directive such as
+              <code>"css"</code>, if the request is a result of processing a CSS
+              <a data-cite="css-values-4"
```
Comment (npm1) [inline index.html:483]: Is this not exported? If it isn't maybe file a bug against that repo?

File: index.html [hunk 4]
```diff
@@ -504,71 +475,72 @@ <h3>
...
+              data-xref-type="css-function">url()</a> directive such as
               <code>@import url()</code> or <code>background: url()</code>;
               [[CSS-VALUES]]
             </li>
             <li>
-              <dfn>"script"</dfn>, if the request is a result of loading any
+              <code>"script"</code>, if the request is a result of loading any
               <a data-cite="HTML#concept-script">script</a> (a classic
```
Comment (npm1) [inline index.html:490]: ditto exported?

File: index.html [hunk 5]
```diff
@@ -1097,7 +1069,7 @@ <h2>
             "PERFORMANCE-TIMELINE-2#dfn-queue-a-performanceentry">Queue</a>
             |entry|.
           </li>
-          <li>Add |entry| to |global|'s <a data-cite=
+          <li>[=Add a PerformanceResourceTiming entry|Add=] |entry| to |global|'s <a data-cite=
           "PERFORMANCE-TIMELINE-2#dfn-performance-entry-buffer">performance
```
Comment (npm1) [inline index.html:1073]: ditto exported?

File: index.html (hunk 1 above)
Comment (yoavweiss) [reply index.html:232]: https://w3c.github.io/resource-timing/#resources-included-in-the-performanceresourcetiming-interface seems to cover it

File: index.html (hunk 2 above)
Comment (yoavweiss) [reply index.html:216]: I agree we can remove it. HTML and Fetch are not specifically defining it, so we probably don't have to

File: index.html (hunk 3 above)
Comment (yoavweiss) [reply index.html:483]: It's exported in both css-values-3 and css-values-4, which seems to create confusion in specref. Talking to @marcoscaceres, this is the syntax we want here..

File: index.html (hunk 4 above)
Comment (yoavweiss) [reply index.html:490]: It's exported, but respec can't deal with `[=script=]`. We can deal with it as a followup.
/cc @marcoscaceres

File: index.html (hunk 5 above)
Comment (yoavweiss) [reply index.html:1073]: Filed https://github.com/w3c/performance-timeline/issues/196. Let's deal with it in a followup, as this is not the only place citing it.

**Review by npm1 — APPROVED** [review]

**Pull request merged**